flashinfer_bench/external/gpumode/setup_volume.py

```
@@ -0,0 +1,49 @@
+    from pathlib import Path
+
+    target = Path(MOUNT_PATH)
+    exists = (target / ".git").exists()
+
+    if exists and not update:
+        print("Volume already populated. Use --update to pull latest changes. Skipping.")
+        return
+
+    if not exists:
+        print("Cloning flashinfer-trace with full LFS...")
+        subprocess.run(["git", "clone", REPO_URL, str(target)], check=True)
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)